packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.test.ts

```diff
@@ -153,4 +167,21 @@ describe('generateCognitoStackResources', () => {
     expect(cognitoStack.userPoolClient!.tokenValidityUnits).toHaveProperty('refreshToken');
     expect(cognitoStack.lambdaConfigPermissions).toHaveProperty('UserPoolPreSignupLambdaInvokePermission');
   });
+
+  it.only('adds correct oidc dependencies', async () => {
```

Comment:
leftover debugging `.only`?

Reply:
we need https://www.npmjs.com/package/eslint-plugin-no-only-tests .

Reply:
I removed it but I guess I didn't push it up